kumascript/macros/CSSSyntax.ejs

```diff
@@ -1,442 +1,324 @@
-        }),
-        anchor: mdn.localString({
-            "en-US": "Brackets",
-            "de"   : "Eckige_Klammern"
-        })
-    },
-    {
-        regexp: /\{(?!\s)|(\d)\}/g,
-        title: mdn.localString({
-            "en-US": "Curly braces: encloses two integers defining the minimal and maximal numbers of occurrences of the entity",
-            "de"   : "Geschweifte Klammern: umschließen zwei Ganzzahlen, die die Minimal- und Maximalanzahl an Vorkommen der Entität angeben"
-        }),
-        anchor: mdn.localString({
-            "en-US": "Curly_braces",
-            "de"   : "Geschweifte_Klammern_(_)"
-        })
-    },
-    {
-        regexp: /!/g,
-        title: mdn.localString({
-            "en-US": "Exclamation point: the group must produce at least one value",
-            "de"   : "Ausrufezeichen: die Gruppe muss mindestens einen Wert erzeugen"
-        }),
-        anchor: mdn.localString({
-            "en-US": "Exclamation_point",
-            "de"   : "Ausrufezeichen_(!)"
-        })
-    },
-];
-
-async function buildLink(match, type) {
-    var link = "/" + locale + "/docs/Web/CSS/";
-    var title = "";
-    var propertyName = type.match(/'(.+?)'/);
-
-    // Handle property references like <'color'>
-    if (propertyName) {
-        if (data.properties[propertyName[1]]) {
-            title = data.properties[propertyName[1]].syntax;
-        }
-        return "<a href=\"" + link + propertyName[1] + "\" title=\"" + title + "\">&lt;" + propertyName[0] + "&gt;</a>";
-
-    // Handle basic types
-    } else if (Object.prototype.hasOwnProperty.call(typeInfo, type)) {
-        var typeLinkName = typeInfo[type].typeLinkName || type;
-
-        link += typeLinkName;
-
-        if (typeInfo[type].title) {
-            title = typeInfo[type].title;
-        } else {
-            title = "";
-        }
-
-    // Handle types which we want to describe using a link to a different page
-    }  else if (Object.prototype.hasOwnProperty.call(data.externallyDescribedTypes, type)) {
-             return "<a href=\"/" + locale + data.externallyDescribedTypes[type].link + "\">&lt;" + type + "&gt;</a>";
-
-    // Handle advanced types having their syntax defined within the 'CSSData' template
-    } else if (data.syntaxes[type]) {
-        return "<a href=\"#" + type + "\">&lt;" + type + "&gt;</a>";
-
-    // Handled advanced types having their syntax not defined
-    } else {
-        return "&lt;" + type + "&gt;";
+    const newValues = parsedWebRef[specName].properties[propertyName].newValues;
+    if (newValues) {
+      newSyntaxes += ` | ${newValues}`;
     }
-
-    return "<a href=\"" + link + "\" title=\"" + title + "\">&lt;" + type + "&gt;</a>";
+  }
+  return syntax;
 }
 
-function addBrackets(str) {
-    return str.match(/\(\)$/) ? str : str + "()";
+/**
+ * Determines the markup to generate for a single node in the AST
+ * generated by css-tree.
+ */
+function renderNode(name, node) {
```

Comment:
Switch parameters to have the most important parameter first:

```suggestion
function renderNode(node, name) {
```

Reply:
Actually I think it has to be like this, because we pass it into css-tree as a decorator: https://github.com/csstree/csstree/blob/HEAD/docs/definition-syntax.md#generatenode-options.